packages/apps/docs/src/components/Layout/components/Header/Header.tsx

```diff
@@ -65,15 +65,15 @@ export const Header: FC<IProps> = ({ menuItems, layout = 'full' }) => {
             href="https://twitter.com/kadena_io"
             title="Go to our X"
           >
-            <SystemIcon.Twitter />
```

Comment:
![image](https://github.com/kadena-community/kadena.js/assets/4015521/5a531ef8-7620-461e-a8cf-8ea4fed57836)

icon sizes seem smaller. is this correct, design wise?

Reply:
checked with isa and its correct @sstraatemans